lib/Util/SVFModule.cpp

```diff
@@ -36,11 +36,32 @@
 #include <llvm/IRReader/IRReader.h>	// IR reader for bit file
 #include <llvm/Support/FileSystem.h>	// for sys::fs::F_None
 
+#include <llvm/IR/IRBuilder.h>
+/*
+
+  svf.main() is used to model the real entry point of a C++ program,
+  which initializes all global C++ objects and then call main().
+
+      define void @svf.main() {
+        entry:
+          call void @_GLOBAL__sub_I_cast.cpp()
+          call void @_GLOBAL__sub_I_1.cpp()
+          call void @_GLOBAL__sub_I_2.cpp()
+          call void bitcast (i32 (i32, i8**)* @main to void ()*)()
```

Comment:
It seems to me you are using main(int argc, char * argv[] , char * env[]) in addSVFMain method, not consistent?

Reply:
It can support the following functions now.
(1) int main()
(2) int main(int argc, char * argv[])
(3) int main(int argc, char * argv[] , char env[])

I forgot to update the comment in Line 50.
Bitcast is not needed any more.